Kino/MainScreen/View/Extensions/SubViews.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  SubViews.swift
```

Comment:
The name "UIView+Extension" would be better here to make it more clear and expandable

Reply:
done